extensions/panache/hibernate-orm-panache/runtime/src/main/java/io/quarkus/hibernate/orm/panache/PanacheEntityBase.java

```diff
@@ -373,6 +373,8 @@ public void flush() {
     /**
      * Find entities matching a query, with optional indexed parameters.
      * This method is a shortcut for <code>find(query, params).stream()</code>.
+     * It requires a transaction to work properly.
```

Comment:
"Requires a transaction to work", and apply to all the same comments in the classes

Reply:
done